### DISCUSSION TITLE
"Like" a model from the results/front page

### DISCUSSION THREAD

**Discussion by namelessentity** [civitai/civitai#486] [Ideas]
At the moment I currently use the "like" button as a "to check out and/or download later" button so I would really appreciate if when you click the little heart that tells you how many likes it currently has it adds it to your favorites list. Maybe it's just me but I do this all the time expecting it to do this and it doesn't. Thanks for your time.

**Comment by Hassam7114** [comment]
I also do the same, I like them to watch or download later, but now there is no option to watch them again.
There should be an option, like in social networks, you like the videos you like and a list of liked videos is created, to which you have access to watch them again, if you wish.
